### DISCUSSION TITLE
Get Community and GitHub friends

### DISCUSSION THREAD

**Discussion by seo-asif** [community/community#59679] [New to GitHub] [answered]
Hii... I am a react developer From Bangladesh.. As I graduated  and post graduated in agriculture... Its not a easy task to become a developer.. Still I am learning.. Feel free to guide some tips to be a good developer....and connect with me plzz...  So I can follow you
my gitHub https://github.com/seo-asif

**Comment by queenofcorgis** [comment]
Hi @seo-asif , we’re glad you’re here 😀

Thanks for sharing your profile with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by seo-asif** [reply, marked as answer]
Ohh..  okay.. Thanks for sharing this info dear ❤️❤️
